### ISSUE TITLE
New commit: Paging for search results - ERROR msg.

### ISSUE THREAD

**Issue by leftside2** [HelloZeroNet/ZeroHello#110]
Site:1HeLLo..Tf3D WebSocket handleRequest error: TypeError: actionFeedSearch() got an unexpected keyword argument 'limit' in UiWebsocket.py line 83 > UiWebsocket.py line 267

(Thanks u for your kind concern on my requesting feature and your efforts, @nofish!)

**Comment by tangdou1** [comment]
you should upgrade your zeronet version to the 3817

**Comment by HelloZeroNet** [comment]
Yeah the version check does not works correctly: https://github.com/HelloZeroNet/ZeroHello/commit/a54339c8afd8e6a9642002547ca35c6726721d61#diff-00c57ccbcf5b14cadd0b1ace2cea4345R120
I will fix when I get back home

**Comment by leftside2** [comment]
@HelloZeroNet  , But it seems like no such a paging feature enabled or added?? Still only limited number of items listed with search keyword 'site:zerome'.

**Comment by tangdou1** [comment]
zerome has no data. you should.search.its hub

**Comment by HelloZeroNet** [comment]
Actually the database is belongs to zerome, so it works. I think @leftside2 is  on pre-Rev3817 version
